packages/plugin/src/rules/no-unused-fields.ts

```diff
@@ -1,10 +1,39 @@
-import { GraphQLSchema, TypeInfo, visit, visitWithTypeInfo } from 'graphql';
+import { FieldDefinitionNode, GraphQLSchema, TypeInfo, visit, visitWithTypeInfo } from 'graphql';
+import { FromSchema } from 'json-schema-to-ts';
+import { GraphQLESTreeNode } from '../estree-converter/types.js';
 import { SiblingOperations } from '../siblings.js';
-import { GraphQLESLintRule } from '../types.js';
+import { GraphQLESLintRule, GraphQLESLintRuleListener } from '../types.js';
 import { requireGraphQLSchemaFromContext, requireSiblingsOperations } from '../utils.js';
 
 const RULE_ID = 'no-unused-fields';
 
+const schema = {
+  type: 'array',
+  maxItems: 1,
+  items: {
+    type: 'object',
+    additionalProperties: false,
+    properties: {
+      ignoredFieldSelectors: {
```

Comment:
A few other names I considered here were:
- `allowedUnusedFields`
- `allowedUnusedFieldSelectors`
- `ignoredFields`

I think `ignoredFieldSelectors` avoids some of the oxymoronic qualities of `allowedUnused*` and provided useful context that this configuration uses ESLint selectors, but am open to other options.

Reply:
I prefer `ignoredFieldSelectors` too